pkg/netpol/connlist/exposed_pods.go

```diff
@@ -0,0 +1,16 @@
+package connlist
+
+import "github.com/np-guard/netpol-analyzer/pkg/netpol/internal/common"
+
+// ExposedPeer captures potential ingress and egress connections data for an exposed Peer
+type ExposedPeer interface {
+	ExposedPeer() Peer // UnprotectedPeer()
```

Comment:
```suggestion
	// ExposedPeer is a peer for which the analysis found some potential exposure info
	ExposedPeer() Peer 
```
We should watch the terms, as "unprotected" and "exposed" are not the same